content/en/docs/ambient/install/platform-prerequisites/index.md

```diff
@@ -47,9 +47,24 @@ and capture pods on the node.
 
 1. If you are using [MicroK8s](https://microk8s.io/), you must append `--set values.cni.cniConfDir=/var/snap/microk8s/current/args/cni-network --set values.cni.cniBinDir=/var/snap/microk8s/current/opt/cni/bin` to the `helm install` command, as MicroK8s [uses nonstandard locations for CNI configuration and binaries](https://microk8s.io/docs/change-cidr).
 
+### K3D
+
+1. If you are using [k3d](https://k3d.io/) - a lightweight wrapper to run [k3s](https://github.com/rancher/k3s), you must append `--set values.cni.cniConfDir=/var/lib/rancher/k3s/agent/etc/cni/net.d --set values.cni.cniBinDir=/bin/` to your `istioctl install` or `helm install` command to install Istio with the `ambient` profile. 
```

Comment:
Believe that the replaced text is still correct, and that this is only true if you are using the default CNI that `k3d` ships with (flannel).

If you were using calico and disabled the default CNI, then the standard paths would be required.

So this should probably still read "if you are using k3s with the default flannel CNI"

Reply:
Thanks, clarified this is for default flannel CNI